config/webpack/loaders/devSiteCssFinder.js

```diff
@@ -0,0 +1,20 @@
+const findCss = (x) => {
+  /**
+  * The regex picks out lines that don't contain // but do contain some form of quotation (' or "),
+  * and an explicit .css or .scss deceleration
+  */
+  const regex = /^(?!\/\/|\/).*["'](.*\.css|.*\.scss)["']/m;
+  const m = regex.exec(x);
+  let fileName;
+
+  if (m !== null) {
+    m.forEach((match, groupIndex) => {
+      if (groupIndex === 1) {
```

Comment:
would the first match index be 0?

Reply:
The first match is the entire import line ex: `var _ActionHeaderDocCommon =_interopRequireDefault(require("./ActionHeaderDocCommon.scss"`, I'll look into making this shorter since there's really no reason to run through both matches.